src/components/ProjectDashboard/components/NftRewardsPanel/RedeemNftsSection/RedeemNftTiles.tsx

```diff
@@ -1,16 +1,16 @@
 import Loading from 'components/Loading'
-import { Jb721DelegateTokensQuery } from 'generated/graphql'
+import { NfTsQuery } from 'generated/graphql'
```

Comment:
**Nit:** capital T?

Reply:
ohh awk. looks like it's just the codegen output making NFTs -> NfTs query. just gonna leave it